xdem/volume.py

```diff
@@ -120,8 +120,17 @@ def interpolate_hypsometric_bins(hypsometric_bins: pd.DataFrame, value_column="v
         bins_under_threshold = bins["count"] < count_threshold
         bins.loc[bins_under_threshold, value_column] = np.nan
 
-    # Interpolate all bins that are NaN.
-    bins[value_column] = bins[value_column].interpolate(method=method, order=order, limit_direction="both")
+    # Count number of valid (finite) values
+    values = bins[value_column]
+    nvalids = len(values[np.isfinite(values)])
```

Comment:
Could this not be simplified to `nvalids = np.count_nonzero(np.isfinite(bins[value_column]))`?

Reply:
Or `bins[value_column].count()` which counts the occurrence of finite values.

Reply:
The last option looks great! The only issue is that by default pandas apparently [does not count inf as a NaN](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.count.html), unless specifically set. So I will stick to the first case, which is clearer.